spread.yaml

```diff
@@ -9,7 +9,6 @@ environment:
     PATH: $GOPATH/bin:/snap/bin:$PATH:/usr/lib/go-1.6/bin
     TESTSLIB: $PROJECT_PATH/tests/lib
     SNAPPY_TESTING: 1
-    SNAP_REEXEC: 0
```

Comment:
We put both our freshly build `snapd` and `snap` into our core snap during the tests. This means that we test re-exec into our freshly-build snapd. Do you think I should add a comment into the prepare.sh to make this clearer?

Ideally we would run all the tests twice, not sure if we want to bear the cost. Maybe we can have a spread-cron test with that?